misk/src/main/kotlin/misk/moshi/MoshiModule.kt

```diff
@@ -13,7 +13,11 @@ import misk.moshi.wire.WireMessageAdapter
 import java.util.Date
 import javax.inject.Singleton
 
-internal class MoshiModule : KAbstractModule() {
+/**
+ * For service setup, prefer to install [misk.MiskCommonServiceModule] than installing [MoshiModule]
+ * directly.
+ */
+class MoshiModule : KAbstractModule() {
```

Comment:
change this to `object MoshiModule` ? It’s unlikely to ever need to be stateful

Reply:
And then you can install it 2x and it doesn’t hurt you!

Reply:
changed!